### PR TITLE
Jakarta EE 11 spec updates

### PR DESCRIPTION
**Pull request by jhanders34**
- Prefix component spec names with Jakarta
- Change Connector spec name to Connectors
- Update application-client xsd to version 11
- Add Jakarta XML Registries to the removed list for EE 9
- Update Full Platform or Full Profile to just Platform
- Remove Enterprise Beans home usage from the specification
- Other small edits for punctuation or spelling errors

### CODE REVIEW

File: specification/src/main/asciidoc/platform/Security.adoc [hunk 1]
```diff
@@ -57,7 +57,7 @@ mentioned in <<a457, Future Directions>>.
 
 The security behavior of a Jakarta EE environment
 may be better understood by examining what happens in a simple
-application with a web client, a JSP user interface, and enterprise bean
+application with a web client, a Jakarta Server Pages user interface, and enterprise bean
```
Comment (anbusampath) [inline specification/src/main/asciidoc/platform/Security.adoc:60]: Shouldn't it call Jakarta Pages?

File: specification/src/main/asciidoc/platform/Security.adoc [hunk 2]
```diff
@@ -842,38 +842,24 @@ Kerberos v5 specification for more details.)
...
-operation of the system. The minimum set of permissions that a Jakarta EE
-product is required to grant to a Jakarta EE application is defined in
-<<a2339, Java Platform, Standard Edition (Java SE) Requirements>>. 
-All Jakarta EE products must be capable
-of deploying application components with exactly these permissions.
-
-A Jakarta EE Product Provider may choose to enable
-selective access to resources using the Java protection model. The
-mechanism used is Jakarta EE product dependent.
-
```
Comment (edburns) [inline specification/src/main/asciidoc/platform/Security.adoc:876]: Great catch. Thanks.

File: specification/src/main/asciidoc/platform/TransactionManagement.adoc [hunk 3]
```diff
@@ -183,18 +183,7 @@ the _service_ method thread, but should not be shared between threads.
...
-in class instance fields. A top-level object is one acquired directly
-from a container managed connection factory object (for example, a JDBC
-_Connection_ acquired from a JDBC _ConnectionFactory_ ), as opposed to
-other objects acquired from these top-level objects (for example, a JDBC
-_Statement_ acquired from a JDBC _Connection_ ). The web container
-ensures that requests to a _SingleThreadModel_ servlet are serialized
-and thus only one thread and one transaction will be able to use the
-object at a time, and that the top-level object will be enlisted in any
-new transaction started by the component.
-* In web components not implementing _SingleThreadModel_ , transactional
```
Comment (edburns) [inline specification/src/main/asciidoc/platform/TransactionManagement.adoc:197]: Great catch.

**Review by edburns — APPROVED** [review]
Thanks very much. Made a few small changes, but otherwise LGTM.

**Pull request merged**